<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choose</class>
 <widget class="QWidget" name="choose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Poke/choose.jpg);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>等级</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTextEdit" name="myLevel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QTextEdit" name="yourName">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="pushButton_5">
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>对手名称</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QTextEdit" name="myName">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QTextEdit" name="info"/>
   </item>
   <item row="3" column="1">
    <widget class="QTextEdit" name="yourLevel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="imok">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>我选好了</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="pushButton_2">
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>名称</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_4">
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>对手等级</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
